src/main/java/frc/robot/subsystems/swerve/ModuleIOTalonFX.java

```diff
@@ -151,61 +182,48 @@ public void updateInputs(ModuleIOInputs inputs) {
         turnAppliedVolts,
         turnCurrent);
```

Comment:
parenthesis on new line? if not, its okay. just a suggestion

Reply:
spotless doesnt like that